amundsen_application/static/js/ducks/allTags/types.ts

```diff
@@ -0,0 +1,22 @@
+import { Tag } from '../../components/Tags/types';
+export { Tag };
```

Comment:
I plan on merging as is since we need more discussion, but I agree that this current pattern should not stay. 

I think these organizational iterations is making it clear that we should move forward with your previous idea of reorganizing the application by related features, which allows for all the types used by the components/containers/ducks to exist in the top level folder for that feature. Problem right now is that it's not immediately clear to me what those different chunks should be. Let me know if you want to discuss more this week so we can close the book on these kinds of changes for a bit.